Source/SessionDelegate.swift

```diff
@@ -184,6 +185,7 @@ open class SessionDelegate: NSObject {
     ///
     /// - returns: The new `SessionDelegate` instance.
     public override init() {
+        self.fileManager = .default
```

Comment:
Take it as a parameter defaulted to `.default`, otherwise you can't really inject it.